pkg/dns/iso3166.go

```diff
@@ -3,262 +3,258 @@ package dns
-	{858, "Uruguay", "UY", "URY"},
-	{860, "Uzbekistan", "UZ", "UZB"},
-	{548, "Vanuatu", "VU", "VUT"},
-	{862, "Venezuela (Bolivarian Republic of)", "VE", "VEN"},
-	{704, "Viet Nam", "VN", "VNM"},
-	{92, "Virgin Islands (British)", "VG", "VGB"},
-	{850, "Virgin Islands (U.S.)", "VI", "VIR"},
-	{876, "Wallis and Futuna", "WF", "WLF"},
-	{732, "Western Sahara*", "EH", "ESH"},
-	{887, "Yemen", "YE", "YEM"},
-	{894, "Zambia", "ZM", "ZMB"},
-	{716, "Zimbabwe", "ZW", "ZWE"},
+	{"AF"},
+	{"AL"},
+	{"DZ"},
+	{"AS"},
+	{"AD"},
+	{"AO"},
+	{"AI"},
+	{"AQ"},
+	{"AG"},
+	{"AR"},
+	{"AM"},
+	{"AW"},
+	{"AU"},
+	{"AT"},
+	{"AZ"},
+	{"BS"},
+	{"BH"},
+	{"BD"},
+	{"BB"},
+	{"BY"},
+	{"BE"},
+	{"BZ"},
+	{"BJ"},
+	{"BM"},
+	{"BT"},
+	{"BO"},
+	{"BQ"},
+	{"BA"},
+	{"BW"},
+	{"BV"},
+	{"BR"},
+	{"IO"},
+	{"BN"},
+	{"BG"},
+	{"BF"},
+	{"BI"},
+	{"CV"},
+	{"KH"},
+	{"CM"},
+	{"CA"},
+	{"KY"},
+	{"CF"},
+	{"TD"},
+	{"CL"},
+	{"CN"},
+	{"CX"},
+	{"CC"},
+	{"CO"},
+	{"KM"},
+	{"CD"},
+	{"CG"},
+	{"CK"},
+	{"CR"},
+	{"HR"},
+	{"CU"},
+	{"CW"},
+	{"CY"},
+	{"CZ"},
+	{"CI"},
+	{"DK"},
+	{"DJ"},
+	{"DM"},
+	{"DO"},
+	{"EC"},
+	{"EG"},
+	{"SV"},
+	{"GQ"},
+	{"ER"},
+	{"EE"},
+	{"SZ"},
+	{"ET"},
+	{"FK"},
+	{"FO"},
+	{"FJ"},
+	{"FI"},
+	{"FR"},
+	{"GF"},
+	{"PF"},
+	{"TF"},
+	{"GA"},
+	{"GM"},
+	{"GE"},
+	{"DE"},
+	{"GH"},
+	{"GI"},
+	{"GR"},
+	{"GL"},
+	{"GD"},
+	{"GP"},
+	{"GU"},
+	{"GT"},
+	{"GG"},
+	{"GN"},
+	{"GW"},
+	{"GY"},
+	{"HT"},
+	{"HM"},
+	{"VA"},
+	{"HN"},
+	{"HK"},
+	{"HU"},
+	{"IS"},
+	{"IN"},
+	{"ID"},
+	{"IR"},
+	{"IQ"},
+	{"IE"},
+	{"IM"},
+	{"IL"},
+	{"IT"},
+	{"JM"},
+	{"JP"},
+	{"JE"},
+	{"JO"},
+	{"KZ"},
+	{"KE"},
+	{"KI"},
+	{"KP"},
+	{"KR"},
+	{"KW"},
+	{"KG"},
+	{"LA"},
+	{"LV"},
+	{"LB"},
+	{"LS"},
+	{"LR"},
+	{"LY"},
+	{"LI"},
+	{"LT"},
+	{"LU"},
+	{"MO"},
+	{"MG"},
+	{"MW"},
+	{"MY"},
+	{"MV"},
+	{"ML"},
+	{"MT"},
+	{"MH"},
+	{"MQ"},
+	{"MR"},
+	{"MU"},
+	{"YT"},
+	{"MX"},
+	{"FM"},
+	{"MD"},
+	{"MC"},
+	{"MN"},
+	{"ME"},
+	{"MS"},
+	{"MA"},
+	{"MZ"},
+	{"MM"},
+	{"NA"},
+	{"NR"},
+	{"NP"},
+	{"NL"},
+	{"NC"},
+	{"NZ"},
+	{"NI"},
+	{"NE"},
+	{"NG"},
+	{"NU"},
+	{"NF"},
+	{"MK"},
+	{"MP"},
+	{"NO"},
+	{"OM"},
+	{"PK"},
+	{"PW"},
+	{"PS"},
+	{"PA"},
+	{"PG"},
+	{"PY"},
+	{"PE"},
+	{"PH"},
+	{"PN"},
+	{"PL"},
+	{"PT"},
+	{"PR"},
+	{"QA"},
+	{"RO"},
+	{"RU"},
+	{"RW"},
+	{"RE"},
+	{"BL"},
+	{"SH"},
+	{"KN"},
+	{"LC"},
+	{"MF"},
+	{"PM"},
+	{"VC"},
+	{"WS"},
+	{"SM"},
+	{"ST"},
+	{"SA"},
+	{"SN"},
+	{"RS"},
+	{"SC"},
+	{"SL"},
+	{"SG"},
+	{"SX"},
+	{"SK"},
+	{"SI"},
+	{"SB"},
+	{"SO"},
+	{"ZA"},
+	{"GS"},
+	{"SS"},
+	{"ES"},
+	{"LK"},
+	{"SD"},
+	{"SR"},
+	{"SJ"},
+	{"SE"},
+	{"CH"},
+	{"SY"},
+	{"TW"},
+	{"TJ"},
+	{"TZ"},
+	{"TH"},
+	{"TL"},
+	{"TG"},
+	{"TK"},
+	{"TO"},
+	{"TT"},
+	{"TN"},
+	{"TR"},
+	{"TM"},
+	{"TC"},
+	{"TV"},
+	{"UG"},
+	{"UA"},
+	{"AE"},
+	{"GB"},
+	{"UM"},
+	{"US"},
+	{"UY"},
+	{"UZ"},
+	{"VU"},
+	{"VE"},
+	{"VN"},
+	{"VG"},
+	{"VI"},
+	{"WF"},
+	{"EH"},
+	{"YE"},
+	{"ZM"},
+	{"ZW"},
```

Comment:
Seems a fairly unnecessary change but ok. Did you look into that module to see if it could be used instead of having to have our own? This was originally added to the repo as i wasn't sure where we were going with the geo stuff at that time. Now we know, we only really need the alpha2 codes for aws (To determine the continent vs country), so just using some other module for that might make more sense.

Reply:
This was failing the linter test so i just removed what wasn't be used. Cool i can look into other modules